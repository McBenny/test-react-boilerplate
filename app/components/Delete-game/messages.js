import { messages as HomeMessages } from '../../containers/HomePage/messages';

export const messages = {
    ...HomeMessages,
    title: 'Delete a game',
    question: 'Are you sure you want to delete the following game?',
    caption: 'Match data',
    match: 'Teams and score',
    warning: 'Warning:',
    warningText: 'This cannot be reverted. Any deleted game is definitely erased. Use with caution!',
    cancel: 'Cancel',
    confirm: 'Confirm'
};
