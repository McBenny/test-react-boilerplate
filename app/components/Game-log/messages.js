export const messages = {
    title: 'Game log:',
    lastAction: 'Last action',
    fullLog: 'Full log'
};
