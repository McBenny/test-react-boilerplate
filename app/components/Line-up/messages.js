export const messages = {
    title: 'Players',
    maxActionsReached: 'Invalid choice for this player',
    listOfPlayers: 'List of players',
    captainInitial: 'C',
    noPlayers: 'No players, please insert players in "Settings"',
    listOfOfficials: 'List of officials',
    noOfficials: 'No officials, please insert officials in "Settings"'
};
