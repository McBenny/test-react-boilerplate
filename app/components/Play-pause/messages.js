export const messages = {
    title: 'Play-pause screen',
    close: 'Close'
};
