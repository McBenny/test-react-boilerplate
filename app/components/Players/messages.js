/**
 * This entire object is embedded in /app/components/Line-up/messages.js
 * Remove keys with extreme care.
 */
export const messages = {
    title: 'Players',
    titles: {
        'boilerplate/Game/ADD_YELLOW_CARD': 'Add a YELLOW card to:',
        'boilerplate/Game/ADD_SUSPENSION': 'Add a 2 minutes suspension to:',
        'boilerplate/Game/ADD_RED_CARD': 'Add a RED card to:',
        'boilerplate/Game/ADD_BLUE_CARD': 'Add a BLUE card to:'
    },
    cancel: 'Cancel',
    maxActionsReached: 'Invalid choice for this player',
    listOfPlayers: 'List of players',
    none: 'none',
    penalty: 'Is it a penalty?',
    captainInitial: 'C',
    goalieInitial: 'GK',
    noPlayers: 'No players, please insert players in',
    listOfOfficials: 'List of officials',
    noOfficials: 'No officials, please insert officials in',
    settings: 'Settings'
};
