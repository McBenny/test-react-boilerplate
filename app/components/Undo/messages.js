export const messages = {
    title: 'Revert an event',
    introduction: 'Do you want to revert the following event?',
    cancel: 'Cancel',
    yes: 'Yes, revert this event'
};
