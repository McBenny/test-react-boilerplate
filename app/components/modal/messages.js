export const messages = {
    cancel: 'Cancel',
    close: 'Close'
};
