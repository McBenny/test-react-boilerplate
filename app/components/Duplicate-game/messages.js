import { messages as HomeMessages } from '../../containers/HomePage/messages';

export const messages = {
    ...HomeMessages,
    title: 'Duplicate a game',
    question: 'Are you sure you want to duplicate the following game?',
    caption: 'Match data',
    match: 'Teams and score',
    warning: 'Warning:',
    warningText: 'This cannot be reverted. Any deleted game is definitely erased. Use with caution!',
    cancel: 'Cancel',
    confirm: 'Confirm'
};
