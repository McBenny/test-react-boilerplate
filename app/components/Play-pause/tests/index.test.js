import React from 'react';
import { render } from '@testing-library/react';

import PlayPause from '../index';
import { messages } from '../messages';

const componentProperties = {
    gameStarted: true,
    gamePaused: false,
    period: 1,
    startHandler: () => {},
    closeHandler: () => {}
};

describe('<PlayPause />', () => {
    it('should render and match the snapshot', () => {
        const {
            container: { firstChild }
        } = render(<PlayPause {...componentProperties} />);
        expect(firstChild).toMatchSnapshot();
    });
});

describe('PlayPause messages', () => {
    it('should be defined and have at least a title', () => {
        expect(messages).toBeDefined();
        expect(messages.title).toBeDefined();
        expect(Object.keys(messages).length).toBeGreaterThan(1);
    });
});
