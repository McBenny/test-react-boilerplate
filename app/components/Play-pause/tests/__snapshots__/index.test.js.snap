// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PlayPause /> should render and match the snapshot 1`] = `null`;
